.github/workflows/ArgoCD-Deploy.yml

```diff
@@ -0,0 +1,31 @@
+# This is a basic workflow to help deploy an application to ArgoCD
+name: ArgoCD-Deploy
+
+# Controls when the workflow will run
+on:
+  workflow_dispatch:
```

Comment:
Hi @abhinavtyagiO
How are you planning to do the release for a particular version and rollout updates via workflows, will you be able to connect with your cluster via github CI?

Reply:
On changing the image in values.yml, argo has an auto-sync policy which brings up pods with the new image based on the rolling strategy. And, yes, we can connect to the cluster if the cluster is public or if the workflow is using a runner that has access to the cluster.

Reply:
Have this tested this before?